data/backend/authentication/urls.py

```diff
@@ -0,0 +1,23 @@
+from . import views
+from .views import register, RegistrationRequestedView, profile
+from django.urls import include, path, re_path
+
+
+urlpatterns = [
+    path("login/", views.LoginView.as_view(), name="login"),
+    path("login.json/", views.LoginView.as_view(), name="login_json"),
+    path("", include("mailauth.urls")),
+    path(
+        "register/success/",
+        RegistrationRequestedView.as_view(),
+        name="registration-requested",
+    ),
+    path("register/", register, name="register"),
+    path("register.json/", register, name="register_json"),
+    re_path(
+        "register/(?P<token>.*)$",
+        views.Activate.as_view(),
+        name="register-activate",
+    ),
+    path("profile.json/", profile, name="profile"),
```

Comment:
I don't see the added value of this confusing `.json/` suffix.

Reply:
It was just to initially differenciate the classical views (made first) with the json views for elm. It's being removed